example/fastify/app.ts

```diff
@@ -0,0 +1,41 @@
+
+// Import Third-party Dependencies
+import fastify from "fastify";
+
+// Import Types
+import { FastifyInstance } from "fastify/types/instance";
+
+// Import Internal Dependencies
+import { webhooksAPI } from "./feature/webhook";
+
+export function buildServer(opts = {}): FastifyInstance {
```

Comment:
Comme ce n'est pas utilisé
```suggestion
export function buildServer(): FastifyInstance {
```